biapol_utilities/label/_match_labels.py

```diff
@@ -1,88 +1,104 @@
 # -*- coding: utf-8 -*-
 
-import numpy as np
 from skimage.segmentation import relabel_sequential
 from ._intersection_over_union import intersection_over_union_matrix
+from ._matching_algorithms import max_similarity
+from ._filter_similarity_matrix import suppressed_similarity
 
-def match_labels_stack(label_stack, method=intersection_over_union_matrix, **kwargs):
-    """Match labels from subsequent slices with specified method
+
+def match_labels_stack(label_stack, **kwargs):
```

Comment:
Does anything speak against adding `metric`, `filter` and `matching` here? If these are mentioned in the docstring, they should also appear here, no?